packages/mobile/src/send/saga.ts

```diff
@@ -223,19 +223,22 @@ export function* buildSendTx(
   comment: string
 ) {
   const contract: Contract = yield call(getERC20TokenContract, tokenAddress)
-  const coreContract: CeloTokenWrapper<any> | undefined = yield call(
-    getTokenContractFromAddress,
-    tokenAddress
-  )
+  const coreContract: Contract = yield call(getStableTokenContract, tokenAddress)
```

Comment:
Using the contract via the abi so new stable tokens work out of the box in the future, with no need to update the code

Reply:
Yes, in this case it would have been fine to just update but now when we add the next stablecoin we don't need to update the SDK anymore, just mark the token as a 'isCoreToken: true` in Firebase, so it's possible to add stablecoins without an app update.

Reply:
I think there are a few more places that could use this behavior to make the stablecoins list be updatable with no code changes, still have hard coded stuff for invites and exchanges (maybe something else?).